tests/attr/test_data_parallel.py

```diff
@@ -454,6 +508,7 @@ def _data_parallel_test_assert(
                     )
                 else:
                     attributions_orig = attr_orig.attribute(**kwargs)
+            self.setUp()
```

Comment:
I added this for determinism - in order to reinitiate the seeds. I'll pass seed to attribute in a separate PR.